README.md

```diff
@@ -131,17 +131,18 @@ $ sudo td-agent-gem install fluent-plugin-geoip
   </record>
 
   # Settings for tag
-  remove_tag_prefix access.
-  tag               geoip.${tag}
+  tag               geoip.${tag[1]}
 
   # To avoid get stacktrace error with `[null, null]` array for elasticsearch.
   skip_adding_null_record  true
 
   # Set log_level for fluentd-v0.10.43 or earlier (default: warn)
   log_level         info
 
-  # Set buffering time (default: 0s)
-  flush_interval    1s
+  <buffer tag>
```

Comment:
To separate chunk, it may need rewrite tag with output plugin not filter plugin, isn't you?